cmd/kepler/main.go

```diff
@@ -78,7 +84,7 @@ func parseArgsAndConfig() (*config.Config, error) {
 	updateConfig := config.RegisterFlags(app)
 	kingpin.MustParse(app.Parse(os.Args[1:]))
 
-	logger := logger.New("info", "text")
+	logger := logger.New("info", "text", os.Stdout)
```

Comment:
this function is using separate logger?

Reply:
yes, because parseArgs needs to log before the "actual / application" logger is created.